specify/specify7
# Use a tooltips library

To test:
- Different places that use tooltips
   - Make sure tooltip is appropriate and useful
   - Make sure tooltip does not overflow the screen
- Tooltips appear after an appropriate amount of time (i.e, after a duration that makes sense)
- Tooltip text can be selected
- Tooltips can be activated with keyboard
- Tooltips can be used on a touchscreen device (mobile phone)
- Disabled elements fallback to browser default tooltips
- Tooltips look nice in dark mode and light mode
- Tooltips have sufficient contrast from the background elements to be easily visible and readable in all places. if this is not good enough, we can consider changing background color or adding some shadow in light mode
- Custom tooltips can be disabled in user preferences (Main -> UI -> Moder tooltips)



Some bugs found so far:

Overlaps with content:
<img width="303" alt="Screenshot 2023-02-16 at 00 11 34" src="https://user-images.githubusercontent.com/40512816/219284520-6f6fad64-c93a-468a-9722-b1ab4818c054.png">

Causes page overflow:
<img width="895" alt="Screenshot 2023-02-16 at 00 11 44" src="https://user-images.githubusercontent.com/40512816/219284565-e60b43b1-b201-4b1c-bbf3-58e42f2b6783.png">
